cairo/scripts/keth.py

```diff
@@ -174,25 +212,32 @@ def e2e(
     block_number: int = typer.Option(
         ..., "-b", "--block", help="Ethereum block number"
     ),
-    compiled_program: Path = typer.Option(
-        Path("build/main_compiled.json"),
-        help="Path to compiled Cairo program",
-        exists=True,
-        dir_okay=False,
-        file_okay=True,
-    ),
     proof_path: Path = typer.Option(
         Path("output/proof.json"),
         help="Path to save proof",
         dir_okay=False,
         file_okay=True,
     ),
     data_dir: Path = typer.Option(
-        Path("data/inputs/1"),
+        Path("data/1/inputs"),
         help="Directory containing prover inputs (ZK-PI)",
         dir_okay=True,
         file_okay=False,
     ),
+    step: Step = typer.Option(
+        Step.MAIN,
+        "-s",
+        "--step",
+        help="Step to run: 'main' or 'init'",
+    ),
+    compiled_program: Path = typer.Option(
+        None,
+        help="Path to compiled Cairo program",
+        exists=True,
+        dir_okay=False,
+        file_okay=True,
+        callback=program_path_callback,
+    ),
```

Comment:
i'd still use `main` as the default program? and overload only if `step` is active?

Reply:
nevermind